### DISCUSSION TITLE
How to define the user-defined equation in New Compound Creater Study

### DISCUSSION THREAD

**Discussion by YW-chemeng** [DanWBR/dwsim#380] [General]
I want to know how to define the user-defined equation in the New Compound Creator Study.
I would appreciate it if anyone could tell me.

![CompCreator 2022-11-13 122131](https://user-images.githubusercontent.com/117691159/201504093-3da91447-6b1e-4742-8505-769b8e67645d.png)

**Comment by DanWBR** [comment]
select User-Defined and enter the equation in the textbox below the dropbox... something like this works:

`0.0032323*Tˆ2+1.43E-4*T+123.23321`
